Source/Order.swift

```diff
@@ -82,7 +82,21 @@ final public class Order: NSObject, Codable {
      Represents a customer in the order
      */
     public class Customer: NSObject, Codable {
+        let id: String
+        let email: String?
```

Comment:
There would need to be 2 different email fields (email and emailSha256). See https://github.com/button/button-merchant-android/pull/15